src/Commands/JobsQueue.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace ActiveCollab\JobQueue\Commands;
+
+use Symfony\Component\Console\Helper\Table;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Exception;
+
+/**
+ * @package ActiveCollab\JobQueue\Commands
+ */
+class JobsQueue extends Command
+{
+    /**
+     * Configure command
+     */
+    protected function configure()
+    {
+        parent::configure();
+
+        $this->setName('jobs_queue')
+             ->setDescription('List all jobs queues grouped by type');
+    }
+
+    /**
+     * @param  InputInterface  $input
+     * @param  OutputInterface $output
+     * @return int
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        try {
+            $connection = $this->getDatabaseConnection($input);
+
+            $type_rows = $connection->execute('SELECT `type`, COUNT(`id`) AS "queued_jobs_count" FROM `jobs_queue` GROUP BY `type`');
```

Comment:
Extend Queue interface with method that returns array where key is job type and value is number of jobs in the queue of that type.